src/routes/gifs/unlike-gif.e2e.test.js

```diff
@@ -0,0 +1,118 @@
+const request = require('supertest')
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+const app = require('../../app')
+
+describe('UNLIKE /gif', () => {
+	let user
+
+	before(async ()=>{
+		 user = await fixtures.insertUser() 
+	})
+	describe('Failure', () => {
+		let accessToken
+		before(async ()=>{
+			const body = { id: user.id, email: user.email }
+			accessToken = await fixtures.generateAccessToken(
```

Comment:
we're not inserting a new user here so you can generate an access token in the first `before` block so it works for both `Failure` and `Success` test cases without regenerating for each of them

Reply:
yes please